es/17_1_Configurando_Libwally.md

```diff
@@ -0,0 +1,191 @@
+make  check-TESTS check-local
+make[3]: Entering directory '/home/standup/libwally-core/src'
+make[4]: Entering directory '/home/standup/libwally-core/src'
+PASS: test_bech32
+PASS: test_psbt
+PASS: test_psbt_limits
+PASS: test_tx
+============================================================================
+Testsuite summary for libwallycore 0.7.8
+============================================================================
+# TOTAL: 4
+# PASS:  4
+# SKIP:  0
+# XFAIL: 0
+# FAIL:  0
+# XPASS: 0
+# ERROR: 0
+============================================================================
+make[4]: Leaving directory '/home/standup/libwally-core/src'
+make[3]: Nothing to be done for 'check-local'.
+make[3]: Leaving directory '/home/standup/libwally-core/src'
+make[2]: Leaving directory '/home/standup/libwally-core/src'
+make[1]: Leaving directory '/home/standup/libwally-core/src'
+make[1]: Entering directory '/home/standup/libwally-core'
+make[1]: Nothing to be done for 'check-am'.
+make[1]: Leaving directory '/home/standup/libwally-core'
+```
+Por último, puede instalar:
+```
+$ sudo make install
+```
+
+## Preparación para Libwally
+
+Entonces, ¿cómo usar Libwally en un programa? Como de costumbre,usted necesitara incluir archivos apropiados y enlazar bibliotecas apropiadas para su código.
+
+### Incluir los archivos
+
+Hay un número considerable de archivos de inclusión posibles:
+```
+$ ls /usr/include/wally*
+/usr/include/wally_address.h  /usr/include/wally_bip39.h   /usr/include/wally_elements.h  /usr/include/wally_script.h
+/usr/include/wally_bip32.h    /usr/include/wally_core.h    /usr/include/wally.hpp	  /usr/include/wally_symmetric.h
+/usr/include/wally_bip38.h    /usr/include/wally_crypto.h  /usr/include/wally_psbt.h	  /usr/include/wally_transaction.h
+```
+Afortunadamente, los nombres de archivo coinciden en gran medida con las secciones de [docs](https://wally.readthedocs.io/en/latest/), por lo que debería ser capaz de incluir los archivos correctos en función de lo que está haciendo, después de incluir el ubicuo `wally_core. h`.
+
+### Enlazar las bibliotecas
+
+También necesitará enlazar las bibliotecas apropiadas:
+```
+$ ls /usr/lib/libsecp* /usr/lib/libwally*
+/usr/lib/libsecp256k1.a   /usr/lib/libwallycore.la  /usr/lib/libwallycore.so. 0
+/usr/lib/libsecp256k1.la  /usr/lib/libwallycore.so  /usr/lib/libwallycore.so. 0.0.0
+```
+Mayormente, usted usara `libwallycore`.
+
+## Configurar un programa Libwally
+
+En comparación con algunas de las bibliotecas anteriores, Libwally es ridículamente fácil de inicializar:
+```
+lw_response = wally_init(0);
+```
+Y cuando termine, hay una práctica función para limpiar cualquier memoria asignada:
+```
+wally_cleanup(0);
+```
+En ambos casos, el argumento es para flags, pero actualmente tiene el valor `0`.
+
+## Prueba un programa de prueba de Libwally
+
+El directorio src contiene [testwally. c](src/16_1_testwally.c), que sólo muestra cómo funcionan las funciones de inicialización y limpieza.
```

Comment:
testwally.c (remove space), also link doesn't work since it's in a lower directory. try ..src/16_1_testwally.c or full path